classes/Task/Restore/NoRestoreFound.php

```diff
@@ -25,20 +25,21 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  */
 
-namespace PrestaShop\Module\AutoUpgrade\Task\Rollback;
+namespace PrestaShop\Module\AutoUpgrade\Task\Restore;
 
 use PrestaShop\Module\AutoUpgrade\Task\AbstractTask;
 use PrestaShop\Module\AutoUpgrade\Task\ExitCode;
+use PrestaShop\Module\AutoUpgrade\Task\TaskName;
 use PrestaShop\Module\AutoUpgrade\Task\TaskType;
 
-class NoRollbackFound extends AbstractTask
+class NoRestoreFound extends AbstractTask
```

Comment:
What do you think about

```suggestion
class RestoreEmpty extends AbstractTask
```

It avoids a negative form.